stubs/pika/pika/spec.pyi

```diff
@@ -451,20 +451,18 @@ class Queue(Class):
         @property
         def synchronous(self): ...
         def decode(self, encoded, offset: int = ...): ...
-        def encode(self): ...
+        def encode(self) -> list[bytes]: ...
 
     class DeclareOk(Method):
         INDEX: ClassVar[int]
-        queue: Incomplete
-        message_count: Incomplete
-        consumer_count: Incomplete
-        def __init__(
-            self, queue: Incomplete | None = ..., message_count: Incomplete | None = ..., consumer_count: Incomplete | None = ...
-        ) -> None: ...
+        queue: str
+        message_count: int
+        consumer_count: int
+        def __init__(self, queue: str, message_count: int, consumer_count: int) -> None: ...
```

Comment:
See comment in stubtest_allowlist. The implementation has `None` defaults for the `__init__` arguments, but that would make instances not work. `encode()` would crash and having a `None` queue is not very useful.